types/prefixfree/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"lib": [
+			"dom",
+			"es5"
+		],
+		"noImplicitAny": true,
+		"noImplicitThis": true,
+		"strictNullChecks": false,
```

Comment:
This should be true to make sure you don't have errors with null-checking when people who use this package have strictNullChecks: true.

Reply:
I accidentally left this as `false` when I copied this `tsconfig.json` from `@types/tape-async`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/919f6b0dc7649e7e3b912d4ddff42f95c6d91a70/types/tape-async/tsconfig.json#L9

Also, a lot of in‑tree packages have `strictNullChecks: false`.